src/Migrations/CoreNodes/dynBaseTypes.cs

```diff
@@ -364,17 +364,17 @@ public static NodeMigrationData Migrate_0630_to_0700(NodeMigrationData data)
         }
     }
 
-    [AlsoKnownAs("Dynamo.Nodes.dynBuildSeq", "Dynamo.Nodes.BuildSeq")]
+    [AlsoKnownAs("Dynamo.Nodes.dynBuildSeq", "Dynamo.Nodes.BuildSeq", "DSCoreNodesUI.NumberRange")]
     public class NumberRange : MigrationNode
     {
-        [NodeMigration(from: "0.6.3.0", to: "0.7.0.0")]
```

Comment:
Search migration test to see if there is any number range node in 0.6.3 and open it :-)